### ISSUE TITLE
ICLR2018 pick up

### ISSUE THREAD

**Issue by shunk031** [shunk031/paper-survey#169]
- Imitation Learning from Visual Data with Multiple Intentions
  - https://openreview.net/forum?id=Hk3ddfWRW
- Mixed Precision Training of Convolutional Neural Networks using Integer Operations
  - https://openreview.net/forum?id=H135uzZ0-
- MaskGAN: Better Text Generation via Filling in the _______
  - https://openreview.net/forum?id=ByOExmWAb
- Unsupervised Representation Learning by Predicting Image Rotations
  - https://openreview.net/forum?id=S1v4N2l0-
- Smooth Loss Functions for Deep Top-k Classification
  - https://openreview.net/forum?id=Hk5elxbRW
- Flipout: Efficient Pseudo-Independent Weight Perturbations on Mini-Batches
  - https://openreview.net/forum?id=rJNpifWAb
- Unsupervised Machine Translation Using Monolingual Corpora Only
  - https://openreview.net/forum?id=rkYTTf-AZ
- Compressing Word Embeddings via Deep Compositional Code Learning
  - https://openreview.net/forum?id=BJRZzFlRb
- Residual Connections Encourage Iterative Inference
  - https://openreview.net/forum?id=SJa9iHgAZ
- On the State of the Art of Evaluation in Neural Language Models
  - https://openreview.net/forum?id=ByJHuTgA-
- Fraternal Dropout
  - https://openreview.net/forum?id=SJyVzQ-C-
- Deep Autoencoding Gaussian Mixture Model for Unsupervised Anomaly Detection
  - https://openreview.net/forum?id=BJJLHbb0-
- Meta-Learning for Semi-Supervised Few-Shot Classification
  - https://openreview.net/forum?id=HJcSzz-CZ
- FearNet: Brain-Inspired Model for Incremental Learning
  - https://openreview.net/forum?id=SJ1Xmf-Rb
- Understanding Deep Neural Networks with Rectified Linear Units
  - https://openreview.net/forum?id=B1J_rgWRW
- Decision Boundary Analysis of Adversarial Examples
  - https://openreview.net/forum?id=BkpiPMbA-
- Deep Active Learning for Named Entity Recognition
  - https://openreview.net/forum?id=ry018WZAZ
- Dynamic Neural Program Embeddings for Program Repair
  - https://openreview.net/forum?id=BJuWrGW0Z
- Hierarchical Representations for Efficient Architecture Search
  - https://openreview.net/forum?id=BJQRKzbA-
- A Compressed Sensing View of Unsupervised Text Embeddings, Bag-of-n-Grams, and LSTMs
 - https://openreview.net/forum?id=B1e5ef-C-
- A New Method of Region Embedding for Text Classification
  - https://openreview.net/forum?id=BkSDMA36Z
- Beyond Word Importance: Contextual Decomposition to Extract Interactions from LSTMs
  - https://openreview.net/forum?id=rkRwGg-0Z
